### PR TITLE
bug: now enableAvatarLODs is acting to changes

### PR DESCRIPTION
**Pull request by AjimenezDCL**
When using the URL_PARAM, the  `KernelConfig`  arrives for the first time with the proper value. 
When using the real feature flag takes, `KernelConfig` is sent with default values while the flag value are retrieved.

AvatarLODS where using only the first value that arrived to `KernelConfig`.

To Test:
Quickest way is to check the settings menu and look for the LODs settings.

LODs should be enabled: https://play.decentraland.zone/?renderer-branch=bug/avatar-lods-feature-flag
LODs should be disabled: https://play.decentraland.org/?renderer-branch=bug/avatar-lods-feature-flag

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs [hunk 1]
```diff
@@ -23,15 +23,33 @@ public class AvatarsLODController : IAvatarsLODController
...
+        {
+            if (enabled == current.features.enableAvatarLODs)
+                return;
+            Initialize(current);
         }
 
         internal void Initialize(KernelConfigModel config)
         {
-            enabled = config.features.enableAvatarLODs;
+            if (enabled == config.features.enableBuilderInWorld)
```
Comment (BrianAmadori) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs:42]: `enableBuilderInWorld`???

Comment (pravusjif) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs:42]: XD WTF

**Review by pravusjif — CHANGES_REQUESTED** [review]
The change works fine but let's address the `enableBuilderInWorld` usage

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs (hunk 1 above)
Comment (AjimenezDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs:42]: Oopsie hahaha

**Review by BrianAmadori — APPROVED** [review]

**Review by pravusjif — APPROVED** [review]

**Pull request merged**